src/libraries/Microsoft.Extensions.Logging.Console/tests/Microsoft.Extensions.Logging.Console.Tests/JsonConsoleFormatterTests.cs

```diff
@@ -507,25 +478,46 @@ private string GetJson(Exception exception)
             return output.ToString();
         }
 
-        [Fact]
-        public void ShouldContainInnerException()
+
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
+        [InlineData(false)]
+        [InlineData(true)]
+        public void ShouldContainInnerException(bool indented)
         {
             Exception rootException = new Exception("root", new Exception("inner"));
             EnsureStackTrace(rootException, rootException.InnerException);
-            string json = GetJson(rootException);
+            string json = GetJson(rootException, indented);
+
             Assert.Contains(rootException.Message, json);
             Assert.Contains(rootException.InnerException.Message, json);
+            
+            Assert.Contains(GetContent(rootException, indented), json);
+            Assert.Contains(GetContent(rootException.InnerException, indented), json);
+        }
+
+        static string GetContent(Exception exception, bool indented)
+        {
+            string indentation = indented ? "\\r\\n" : " ";
+            return exception.ToString()
+                .Replace("\\", "\\\\")
```

Comment:
needed for the stacktrace content that uses path `C:\\X\\Y\\Z` in the json for the path `C:\X\Y\Z`.